src/main/resources/assets/toomanyorigins/lang/tr_tr.json

```diff
@@ -4,58 +4,71 @@
+    "power.toomanyorigins.drainage.name": "Emiş",
+    "power.toomanyorigins.drainage.description": "Bir düşman öldürdüğünde yaşam enerjisini emersin, bu açlığını doldurur.",
+    "power.toomanyorigins.hell_raised.name": "Cehennemli",
+    "power.toomanyorigins.hell_raised.description": "Oklar için alternatif bir tarife erişimin var.",
+    "power.toomanyorigins.spirit_strider.name": "Ruh Gezer",
+    "power.toomanyorigins.spirit_strider.description": "Her zaman bir seviye ekstra ruh hızın var.",
+    "power.toomanyorigins.heterotroph.name": "Heterotrof",
+    "power.toomanyorigins.heterotroph.description": "Yemek eşyaları senin için daha az doyurucu.",
+
     "toomanyorigins.midnightconfig.title": "TooManyOrigins Ayarları",
     "toomanyorigins.midnightconfig.dragonFireballMixinOptions": "§nEnder Ejderhası Ateş Topu Karışım Ayarları§r",
     "toomanyorigins.midnightconfig.shouldFireballDamageUndead": "Ateş Topu Ölü Yaratıklara Hasar Verir",
   
     "toomanyorigins.gui.version_mismatch": "Bu server başka bir TooManyOrigins versiyonu (v%1$s) çalıştırıyor ve sende mevcut olan versiyonla uyumlu değil (v%2$s).",
     "toomanyorigins.content.disabled": "Devre Dışı",
     "toomanyorigins.content.disabled_message": "Eski TooManyOrigins modülü '%1$s' devre dışı. 'data/toomanyorigins/legacy_content.json' dosya konumunda bulunan bir datapack ile etkinleştiriniz.",
+    "dataPack.legacytoomanyorigins.description": "Bütün rework öncesi TooManyOrigins içeriklerini geri getirir.",
 
-    "origin.legacytoomanyorigins.dragonborn.description": "Yarı İnsan, Yarı Ejderha. Vahşi nefesleri ve yapıları Son'un diğer sakinlerinin onlara baş eğmelerini sağlar.",
+    "origin.legacytoomanyorigins.dragonborn.description": "Yarı İnsan, Yarı Ejderha. Vahşi nefesleri ve yapıları End'in diğer sakinlerinin onlara baş eğmelerini sağlar.",
 
     "power.legacytoomanyorigins.dragon_breath.name": "Ejderha Nefesi",
     "power.legacytoomanyorigins.dragon_breath.description": "Zırh Delen Ejderhası Ateşi olan ve çarptığı yerde kısa süreliğine kalıcı bir bulut bırakan bir top atabilirsin.",
-    "power.legacytoomanyorigins.no_knockback.name": "Sabit Duruş",
-    "power.legacytoomanyorigins.no_knockback.description": "Düşmanlarına karşı güç ve gururunla durursun, bu sayede saldılarından savrulmassın.",
     "power.legacytoomanyorigins.light_up.name": "Wyrmsmith'in Dokunuşu",
-    "power.legacytoomanyorigins.light_up.description": "Fırınları, kamp ateşlerini ve iksir standlarını nefesinle yakabilirsin.",
-    "power.legacytoomanyorigins.loose_scales.name": "İnce Deri",
-    "power.legacytoomanyorigins.loose_scales.description": "Arkandaki ince derinden dolayı arkandan daha fazla hasar alırsın.",
+    "power.legacytoomanyorigins.light_up.description": "Fırınları, kamp ateşlerini ve iksir standlarını bir tek nefesinle yakabilirsin.",
+    "power.legacytoomanyorigins.no_knockback.name": "Hacıyatmaz",
```

Comment:
Stole the idea of Hacıyatmaz from the old translator because I kinda like it, just google "Hacıyatmaz" to see why